### ISSUE TITLE
Workspace settings not staying in workspace json file

### ISSUE THREAD

**Issue by codeunifier** [aaronpowell/vscode-profile-switcher#35]
Adding a "profileSwitcher.storage" or "profileSwitcher.extensions" block to the workspace json file will update the main settings.json when switching to that profile. I would expect profileSwitcher settings set within in a workspace to only update the workspace json, not the main settings.json.

Doing so would allow someone to have multiple workspaces open in different windows with different profiles / themes / settings set for each window. Since this extension always edits the settings.json, it updates all open windows.

**Comment by aaronpowell** [comment]
You're right on the current behaviour, it's doing system-wide changes to VS Code, and I'm assuming that's where you're going with #36.

It should be possible to do workspace-specific settings through modifying the workspace `.vscode/settings.json` file, but extensions won't be possible, as extensions are managed within the VS Code host (the host OS or a remote host like Codespaces or WSL).
